setup.cfg

```diff
@@ -54,10 +53,6 @@ invenio_access.system_roles =
     any_user = invenio_access.permissions:any_user
     authenticated_user = invenio_access.permissions:authenticated_user
     system_process = invenio_access.permissions:system_process
-invenio_admin.views =
```

Comment:
These should not be removed as invenio-access is used by other invenio instances e.g cds-videos. If we remove this, upgrading in the latest invenio-access will remove the admin panel  views without providing an alternative at the moment. I would wait until invenio-access provides the same implementation utilizing invenio-administration. @kpsherva wdyt?

Reply:
we had a discussion with Lars in Münster about this, thats the reason why i created that PR. But your point is a valid point to not remove invenio-admin at the moment. let's wait for other opinions.

Reply:
does this argument also apply to other packages?